HiP-FeatureToggle/Controllers/FeatureGroupsController.cs

```diff
@@ -2,28 +2,28 @@
 using PaderbornUniversity.SILab.Hip.FeatureToggle.Managers;
 using PaderbornUniversity.SILab.Hip.FeatureToggle.Models.Entity;
 using PaderbornUniversity.SILab.Hip.FeatureToggle.Models.Rest;
-using PaderbornUniversity.SILab.Hip.FeatureToggle.Services;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using Microsoft.AspNetCore.Authorization;
+using PaderbornUniversity.SILab.Hip.FeatureToggle.Data;
 
 namespace PaderbornUniversity.SILab.Hip.FeatureToggle.Controllers
 {
     /// <summary>
     /// Provides methods to add/remove feature groups and to assign users to these groups.
     /// </summary>
+    [Authorize]
```

Comment:
I think we should set this to `Authorize("read:featuretoggle")`, so all calls to this API require the `"read:featuretoggle"` scope. Additionally, for all calls that require write privileges, the `"write:featuretoggle"` scope should be required (see below)